cmake/CxxQt.cmake

```diff
@@ -37,12 +37,27 @@ function(cxx_qt_import_crate)
     endif()
   endif()
 
+  if (NOT DEFINED IMPORT_CRATE_QT_MODULES)
+    message(FATAL_ERROR "At least one Qt module must be specified")
```

Comment:
should we always require at least one?

Note how in CxxQtBuilder at the moment it always defaults to adding qt_modules

Reply:
(and should we allow for using outside of Qt, eg no Qt modules etc)

Reply:
done